### DISCUSSION TITLE
pluripotent cells  (BFF in microservices)

### DISCUSSION THREAD

**Discussion by nphias** [holochain/holochain#3080] [Ideas]
I am interested in keeping backend logic on the holochain side more than having it go out beyond the websocket into frontend logic..  in web2 Microservices this would be done using the BFF pattern. 
So in holochain this would take the form of a type of cell that coordinates between other cells.. and has the power to clone cells into existence

**Comment by jost-s** [comment]
Hi Thomas, Is your suggestion to enable cloning in zome code or does it go beyond that?

**Reply by nphias** [reply]
yes basically as @mattyg was saying .. increasing the HDK function feature list inside the conductor wasm host so that we can have more control there about intercell comms and creation etc..  from the meeting, people didnt think it was a big reach.. so i could make a feature request issue for it and hopefully it would go into 0.3x

**Reply by maackle** [reply]
Sounds closely related to #2993

Also, the idea of [having coordinator zomes which coordinate across multiple cells](https://github.com/holochain/holochain/discussions/2167#discussioncomment-5513713) has been suggested (but perhaps not enough has been written), which I think would cover the "cell that coordinates between other cells" part
